utils/dry-run-deploys/src/main.rs

```diff
@@ -48,8 +49,11 @@ async fn main() -> Result<(), anyhow::Error> {
     let chain_download_path = normalize_path(&opts.chain_download_path)?;
     let lmdb_path = normalize_path(&opts.lmdb_path)?;
 
+    let merkle_tree_hash_activation = EraId::from(0u64);
```

Comment:
I think eventually tools like this should just read a chainspec. Not in this PR, but maybe we should ticket it and add a `TODO` comment.

Reply:
TODO added [here](https://github.com/casper-network/casper-node/pull/2531/commits/2ddb40a696aa96983c34e0a7e1ed31b7c67d0570).
Resolving for now due to https://github.com/casper-network/casper-node/pull/2531#discussion_r778823863